Tests/SwiftPlotTests/Annotation/annotation-arrow.swift

```diff
@@ -11,46 +12,32 @@ import QuartzRenderer
 extension AnnotationTests {
 
   func testAnnotationArrow() throws {
-    
+
+    //TODO: update reference image
+
     let fileName = "_30_arrow_annotation"
 
-    let x:[Float] = [10,100,263,489]
-    let y:[Float] = [10,120,500,800]
+    func function(_ x: Float) -> Float {
+      return sin(x)
+    }
 
     let lineGraph = LineGraph<Float, Float>(enablePrimaryAxisGrid: true)
-    lineGraph.addSeries(x, y, label: "Plot 1", color: .lightBlue)
-    lineGraph.plotTitle.title = "SINGLE SERIES"
+    lineGraph.addFunction(function, minX: -5.0, maxX: 5.0, label: "sin(x)", color: .lightBlue)
```

Comment:
Here instead of wrapping `sin` in `function` and using that, you can use `sin` directly as the first parameter since it has the same type signature.